pytorch_lightning/strategies/strategy.py

```diff
@@ -180,10 +180,6 @@ def optimizer_step(
         model = model or self.lightning_module
         self.precision_plugin.optimizer_step(model, optimizer, opt_idx, closure, **kwargs)
 
-    def optimizer_zero_grad(self, current_epoch: int, batch_idx: int, optimizer: Optimizer, opt_idx: int) -> None:
-        """Zeros all model parameter's gradients."""
-        self.lightning_module.optimizer_zero_grad(current_epoch, batch_idx, optimizer, opt_idx)
-
```

Comment:
We could again split this out to another PR for discussion to ensure this PR does only one thing

Reply:
Done